.shiprc

```diff
@@ -0,0 +1,3 @@
+{
+    "files": {}
```

Comment:
The whole file can be omitted, if it's a JS lib and the you're using the defaults.